### PR TITLE
Add documentation for running Bazel in an airgapped environment using distribution directories

### PR DESCRIPTION
**Pull request by jin**

### CODE REVIEW

File: site/docs/guide.md [hunk 1]
```diff
@@ -327,20 +327,77 @@ be determined, for example to manually clean up the cache. The cache is never
...
+bazel build @additional_distfiles//:archives.tar
+```
+
+Export this tarball to a directory that can be copied into your airgapped
+environment. Note the `--strip-components` flag, because `--distdir` can be
+quite finicky with the directory nesting level:
+
+
+```
+tar xvf bazel-bin/external/additional_distfiles/archives.tar -C
```
Comment (aiuto) [inline site/docs/guide.md:390]: Perhaps split on two lines with a \\ at the end so it won't horizontal scroll and is still pastable.

Also, if you move --strip-compontns before -C NEWDIRECTORY, then someone can copy-paste the entire preamble and just type the distdir path.

Comment (jin) [reply site/docs/guide.md:390]: Resolving in the internal CL: cr/313751874

**Comment by aiuto** [comment]
Thanks for that write-up.  ISTM that what we could use is a bazel mode where you can run a build and get a simple plain text file of each file we downloaded and the checksum.
Then someone could wget/curl each to their own distdir without needed bazel sources - which is just sort of ridiculous.

**Comment by jin** [comment]
@aiuto We have that in some capacity with `bazel sync` and workspace resolved files.

File: site/docs/guide.md [hunk 2]
```diff
@@ -327,20 +327,77 @@ be determined, for example to manually clean up the cache. The cache is never
...
+"$NEW_DIRECTORY" --strip-components=3
+```
+
+Finally, when you use Bazel in your airgapped environment, pass the `--distdir`
+flag pointing to the directory. For convenice, you can add it as an `.bazelrc`
```
Comment (gregestren) [inline site/docs/guide.md:395]: typo nit

**Comment by aiuto** [comment]
> @aiuto We have that in some capacity with `bazel sync` and workspace resolved files.

Yes. But the implementation is inconvenient and hard to use. But I don't want to derail this PR with that discussion.

**Comment by jin** [comment]
No worries, and I agree with your general idea. It would be useful for
audit purposes.

**Pull request closed without merging**